doc/tutorials/core/univ_intrin/univ_intrin.markdown

```diff
@@ -269,24 +271,67 @@ We will now look at the vectorized version of 1-D convolution.
 1. In our case, the kernel is a float. Since the kernel's datatype is the largest, we convert src to float32, forming *src_32*. We also make a border like we did for the naive case.
     @snippet univ_intrin.cpp convolution-1D-convert
 
-2. We create an array(called kernel wide) of `v_float32` of size *ksize*. Each row of the kernel_wide array contains one value of the kernel spread across the register.
-    @snippet univ_intrin.cpp convolution-1D-kernel
-
-3. Now, for each column *i* in the *src* vector, we need to calculate the sum of the product of the corresponding window around *i*.
+2. Now, for each column row in the *kernel*, we calculate the element wise product of all windows of length `step`. We add these values to the already stored values in ans
```

Comment:
Do you mean `column and row`?

Reply:
I actually meant each column. I'll make the change